core/debugger/remote_debugger.cpp

```diff
@@ -761,6 +761,12 @@ void RemoteDebugger::debug(bool p_can_continue, bool p_is_error_breakpoint) {
 					frame.file = script_lang->debug_get_stack_level_source(i);
 					frame.line = script_lang->debug_get_stack_level_line(i);
 					frame.func = script_lang->debug_get_stack_level_function(i);
+
+					ScriptInstance *instance = script_lang->debug_get_stack_level_instance(i);
+					Object *owner = instance->get_owner();
+					Node *node = Object::cast_to<Node>(owner);
+					frame.owner = node->get_filename();
```

Comment:
Can there be situations where either `instance` or `node` end up as null pointers, which would crash?

Reply:
I found one example which suggest that sometimes instance could be nullptr. https://github.com/godotengine/godot/blob/master/core/debugger/remote_debugger.cpp#L775, sounds like that there is should be null pointer check.